woocommerce/google-listings-and-ads
# Fix table title and icon button spacing

### Changes proposed in this Pull Request:

Closes #529 .

This PR fixes the title and button layout issue in `TableCard` component. It adds a 16px margin spacing between title and button.

This affects a few places:

- https://gla1.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fdashboard : Dashboard page - Programs table (The "Add paid campaign" button is aligned to the right)
- https://gla1.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Freports%2Fprograms : Programs report
- https://gla1.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Freports%2Fproducts : Products report
- https://gla1.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fproduct-feed : Product Feed page - Issues to resolve table, and Product feed table.

### Screenshots:

Before: 

![image](https://user-images.githubusercontent.com/417342/117053545-abb74800-ad4b-11eb-8f23-0d36f4aa6d36.png)

After (some examples):

![image](https://user-images.githubusercontent.com/417342/117052708-a0175180-ad4a-11eb-918b-1a9351525a37.png)

![image](https://user-images.githubusercontent.com/417342/117052791-bb825c80-ad4a-11eb-94ae-594bfc21d224.png)

### Detailed test instructions:

1. Open the affected URLs mentioned above.
2. Verify that there is some spacing between the table title and the buttons, they are not cluttered together.

### Changelog Note:

Add spacing between title and action buttons in TableCard.
